utils/getClaimStatus.tsx

```diff
@@ -2,22 +2,65 @@
  * Utility file to get content for the Claim Status section.
  */
 
-import { ClaimStatusContent, I18nString } from '../types/common'
+import claimStatusJson from '../public/locales/en/claim-status.json'
+import { ClaimStatusContent, I18nString, TransLineProps } from '../types/common'
 import { ScenarioType } from './getScenarioContent'
+import getUrl, { UrlType } from './getUrl'
 
 /**
- * Get Claim Status description content.
+ * Helper function to get shared Claim Status translation string prefix.
  */
-export function getClaimStatusDescription(scenarioType: ScenarioType): I18nString {
-  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}.description`
+function getTranslationPrefix(scenarioType: ScenarioType): I18nString {
+  return `claim-status:scenarios.${ScenarioType[scenarioType].toLowerCase()}`
+}
+
+/**
+ * Get Claim Status heading.
+ */
+export function getClaimStatusHeading(scenarioType: ScenarioType): I18nString {
+  return getTranslationPrefix(scenarioType) + '.heading'
+}
+
+/**
+ * Get Claim Status summary.
+ *
+ * Note: This function has a lot of verbosity due to Typescript + eslint.
+ */
+export function getClaimStatusSummary(scenarioType: ScenarioType): TransLineProps {
+  const transLineProps: TransLineProps = {
+    i18nKey: getTranslationPrefix(scenarioType) + '.summary.text',
+  }
+  // Explicitly initialize to an empty array.
+  transLineProps.links = []
+
+  // Explicitly cast the scenarioType into one of the keys of claim-status.json
+  // (i.e. scenario1 | scenario2 etc)
+  const scenarioString = ScenarioType[scenarioType].toLowerCase() as keyof typeof claimStatusJson.scenarios
```

Comment:
Also, if you are like me and ever struggled with `keyof typeof`, here is a very good [stackoverflow  on it](https://stackoverflow.com/a/62764510).